src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -22,6 +22,9 @@ public class DeleteCommandParser implements Parser<DeleteCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public DeleteCommand parse(String args) throws ParseException {
+        if (args.trim().isEmpty()) {
```

Comment:
Are you changing the behaviour of the delete command here? There are some changes that are disallowed by feature freeze.